PWR/PWR_RTC/Core/Src/main.c

```diff
@@ -0,0 +1,97 @@
+#include "stm32l053xx.h"
+
+//------ DEFINES
+//--- INPUTS
+#define USR_BT_PRESS		(!(GPIOA->IDR & GPIO_IDR_ID10_Msk))
+
+//--- OUTPUTS
+#define GREEN_LED_ON		(GPIOA->ODR |= GPIO_ODR_OD7_Msk)
+#define	GREEN_LED_OFF		(GPIOA->ODR &= ~GPIO_ODR_OD7_Msk)
+#define TOGGLE_GREEN_LED	(GPIOA->ODR ^=GPIO_IDR_ID7_Msk)
+
+//--- GENERAL
+#define RTC_30_SECONDS	32000	//used for RTC config
+#define RTC_KEY1		0xCA	//1st key to unlock RTC's registers (write to RTC_WRP)
+#define RTC_KEY2		0x53	//2nd key to unlock RTC's registers (write to RTC_WRP)
+
+int main(void)
+{
+	/*							   *
+	 * --- GPIO INITIALIZATION --- *
+	 *							   */
+	/* -> GPIOA
+	 * PA14 -> AF mode (SWD_IO)		| Low speed	| Push pull
+	 * PA13 -> AF mode (SWD_CK)		| Low speed	| Push pull

```

Comment:
Where are you using PA14 and PA13? If you're not using, remove the comments.

Reply:
They're used by the ST-link, for programming and debugging. It's more like a reminder of that, to not try to use them for other functions, and I've put this comment because you or Vini asked to in the EXTI example

Reply:
Ok!